test/support/analyzer_verification/assert_not_call.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ElixirAnalyzer.Support.AnalyzerVerification.AssertNoCall do
```

Comment:
Nitpick: filename doesn't match the module name, `_not_` -> `_no_`, same applies to the file `test/elixir_analyzer/exercise_test/assert_not_call_test.exs`